udf-compiler/src/test/scala/com/nvidia/spark/OpcodeSuite.scala

```diff
@@ -36,6 +36,7 @@ class OpcodeSuite extends AnyFunSuite {
   val conf: SparkConf = new SparkConf()
     .set("spark.sql.extensions", "com.nvidia.spark.udf.Plugin")
     .set("spark.rapids.sql.udfCompiler.enabled", "true")
+    .set("spark.sql.ansi.enabled", "false")
```

Comment:
Curious why we need to disable ANSI here?  I assume this is for Spark 4.0, but then I would expect there to be corresponding negative tests where we are testing that we do _not_ replace UDFs because of ANSI expression semantics.  Seems like we need a comment and/or pointer to a tracking issue here.

Reply:
correct that is due to spark 4, let me file something at least with the failures I see with ansi enabled.

Reply:
https://github.com/NVIDIA/spark-rapids/issues/11633